opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -241,16 +241,6 @@ message Sample {
   // result has a list of values that is the element-wise sum of the
   // lists of the originals.
   repeated int64 value = 2;
-  // label includes additional context for this sample. It can include
-  // things like a thread id, allocation size, etc.
-  //
-  // NOTE: While possible, having multiple values for the same label key is
-  // strongly discouraged and should never be used. Most tools (e.g. pprof) do
-  // not have good (or any) support for multi-value labels. And an even more
-  // discouraged case is having a string label and a numeric label of the same
-  // name on a sample.  Again, possible to express, but should not be used.
-  // [deprecated, superseded by attributes]
-  repeated Label label = 3;
```

Comment:
Similar to https://github.com/open-telemetry/opentelemetry-proto/pull/578#discussion_r1736133541, indexing of fields in messages will be updated once before the profiling signal becomes stable.